contributing.md

```diff
@@ -169,16 +181,12 @@ Generates a changefile based on the dev branch. This should be run on your PR be
 
 Checks that changefiles have been generated for relevant changed packages. Used by the CI tool on PRs, can also be used locally.
 
-- `npm run beachball:publish`
+- `npm run beachball:bump`
 
-Bumps versions and publishes all packages that have relevant changefiles. If the changefiles types are set to `None`, no version bump or publish is done. Run on push to the `master` branch.
+Bumps versions for packages with relevant changefiles. Run on `master` branch to check which packages are slated for release.
```

Comment:
Maybe add a note that this should not be done when submitting a PR?

Reply:
Good point, will add it.

Reply:
If we want the PR number in the change file message, we should suggest they should first create the PR, then run beachball:change, add the PR number in the message and then push to their feature branch again?

Reply:
I've added a comment for the PR number in the description for the `beachball:change` command.